modules/apm/src/main/java/org/elasticsearch/telemetry/apm/internal/metrics/APMMeter.java

```diff
@@ -0,0 +1,190 @@
+        return instruments.registerDoubleCounter(name, description, unit);
+    }
+
+    @Override
+    public DoubleCounter getDoubleCounter(String name) {
+        return instruments.getDoubleCounter(name);
+    }
+
+    @Override
+    public DoubleUpDownCounter registerDoubleUpDownCounter(String name, String description, String unit) {
+        return instruments.registerDoubleUpDownCounter(name, description, unit);
+    }
+
+    @Override
+    public DoubleUpDownCounter getDoubleUpDownCounter(String name) {
+        return instruments.getDoubleUpDownCounter(name);
+    }
+
+    @Override
+    public DoubleGauge registerDoubleGauge(String name, String description, String unit) {
+        return instruments.registerDoubleGauge(name, description, unit);
+    }
+
+    @Override
+    public DoubleGauge getDoubleGauge(String name) {
+        return instruments.getDoubleGauge(name);
+    }
+
+    @Override
+    public DoubleHistogram registerDoubleHistogram(String name, String description, String unit) {
+        return instruments.registerDoubleHistogram(name, description, unit);
+    }
+
+    @Override
+    public DoubleHistogram getDoubleHistogram(String name) {
+        return instruments.getDoubleHistogram(name);
+    }
+
+    @Override
+    public LongCounter registerLongCounter(String name, String description, String unit) {
+        return instruments.registerLongCounter(name, description, unit);
+    }
+
+    @Override
+    public LongCounter getLongCounter(String name) {
+        return instruments.getLongCounter(name);
+    }
+
+    @Override
+    public LongUpDownCounter registerLongUpDownCounter(String name, String description, String unit) {
+        return instruments.registerLongUpDownCounter(name, description, unit);
+    }
+
+    @Override
+    public LongUpDownCounter getLongUpDownCounter(String name) {
+        return instruments.getLongUpDownCounter(name);
+    }
+
+    @Override
+    public LongGauge registerLongGauge(String name, String description, String unit) {
+        return instruments.registerLongGauge(name, description, unit);
+    }
+
+    @Override
+    public LongGauge getLongGauge(String name) {
+        return instruments.getLongGauge(name);
+    }
+
+    @Override
+    public LongHistogram registerLongHistogram(String name, String description, String unit) {
+        return instruments.registerLongHistogram(name, description, unit);
+    }
+
+    @Override
+    public LongHistogram getLongHistogram(String name) {
+        return instruments.getLongHistogram(name);
+    }
+
+    Meter createApmServices() {
+        assert this.enabled;
+
+        SetOnce<Meter> provider = new SetOnce<>();
+        AccessController.doPrivileged((PrivilegedAction<Void>) () -> {
+            provider.set(otelMeterSupplier.get());
+            return null;
+        });
+        return provider.get();
+    }
+
+    private void destroyApmServices() {
```

Comment:
we could refactor this method to return noop instance. 
(same as above createApmServices)

Reply:
Thanks for adding a commit to do this.